baseband/vdif/tests/test_vdif.py

```diff
@@ -983,6 +984,40 @@ def test_io_invalid(self, tmpdir):
             vdif.open(tmp_file, 's')
 
 
+def test_sequentialfile(tmpdir):
+    """Tests writing and reading of sequential files.
+
+    These tests focus on reading and writing with templates.
+    """
+
+    # Use sample file as basis of a file sequence.
+    with vdif.open(SAMPLE_FILE, 'rs') as fh:
+        header = fh.header0.copy()
+        data = fh.read()
+        dtime = fh.stop_time - fh.start_time
+    data = np.r_[data, data, data]
+
+    # Create a file sequence using template.
+    template = str(tmpdir.join('f.{file_nr:03d}.vdif'))
+    files = sf.FileNameSequencer(template, {})
```

Comment:
Does the `FileNameSequencer` need the empty dict? I think that should be the default...

Reply:
Changed (for DADA and GUPPI subclasses as well).  Let me know if you want the examples and code to explicitly pass ``header=header...``.